src/eth/TransactionObject.js

```diff
@@ -116,7 +118,9 @@ export default class TransactionObject extends TransactionLifeCycle {
       this.setMined();
     } catch (err) {
       this._errorMessage = err.message;
-      this.setError(err);
+      console.error(err);
+      this._nonceService.setCounts();
+      this.setError();
```

Comment:
regression here (maybe it happened in the merge?) -- need to continue to pass `err` as an argument